com.unity.perception/Runtime/GroundTruth/Labelers/KeypointLabeler.cs

```diff
@@ -423,6 +624,12 @@ void ProcessLabel(Labeling labeledEntity)
                     }
                 }
 
+                var occlusionOverrider = labeledEntity.GetComponent<KeypointOcclusionOverrides>();
```

Comment:
This should probably be `GetComponentInParent`, so that setting it on higher objects in hierarchy affects the scale

Reply:
makes sense. changed